bin/pygrb/pycbc_pygrb_page_tables

```diff
@@ -48,6 +48,26 @@ __program__ = "pycbc_pygrb_page_tables"
 # =============================================================================
 # Functions
 # =============================================================================
+def additional_injection_data(data, ifos):
+    """Calculates chirp mass and effective sistance"""
+    data['mchirp'] = mchirp_from_mass1_mass2(data['mass1'],
+                                            data['mass2'])
```

Comment:
You may need an extra space here.  See what code climate says :-)

Reply:
Code climete is not complaining, so far...